### PR TITLE
feat: Upgrade to Babel 6

### PR DESCRIPTION
**Pull request by doctyper**
This pull request upgrades Wildcat to jspm 0.16.34 + Babel 6

Test Plan: `make install`

### CODE REVIEW

File: packages/react-wildcat/src/staticServer.js [hunk 1]
```diff
@@ -213,9 +215,11 @@ function start() {
 }
 
 function close() {
-    if (server && server.close) {
-        return server.close();
+    if (!server || !server.close) {
```
Comment (jamsea) [inline packages/react-wildcat/src/staticServer.js]: Why the change from the && to the  ||? What case is this catching that the previous method wasn't? It seemed like the previous check with && was a lot nicer

**Comment by jamsea** [comment]
I'm guessing you want to squash this and fix the merge conflicts but other than that it looks good to me

File: packages/react-wildcat/src/staticServer.js (hunk 1 above)
Comment (doctyper) [inline packages/react-wildcat/src/staticServer.js]: I try to return the happy path at the end of the method, and bail at the earliest possible moment.
